build/repo.targets

```diff
@@ -274,4 +274,63 @@
            Text="Repository %(Repository.Identity) is missing NuGetPackageVerifier.json. Expected file to exist in %(Repository.RootPath)NuGetPackageVerifier.json" />
   </Target>
 
-</Project>
+  <Target Name="DoCodeSigning" DependsOnTargets="_SetupCodeSign;CodeSign;_CopySignedFilesToArtifacts" />
+
+  <Target Name="_SetupCodeSign">
+    <PropertyGroup>
+      <!-- Make sure we're not disabling code signing -->
+      <DisableCodeSigning>false</DisableCodeSigning>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <!-- We're passing a custom list of files - clear out the exclusion list so it doesn't interfere -->
+      <FilesToExcludeFromSigning Remove="@(FilesToExcludeFromSigning)" />
+
+      <!-- Make sure 3rd party binaries get 3rd party certificate -->
+      <CustomFileSignInfo Include="MsgPack.dll" CertificateName="3PartySHA2" />
```

Comment:
How was this list generated? I just want to double check but it looks fine.

Reply:
I manually looked at the contents of all of the .nupkg's/.zips that we're signing, and cross-referenced with what's in master: https://github.com/dotnet/aspnetcore/blob/master/eng/Signing.props